.circleci/config.yml

```diff
@@ -1281,7 +1189,7 @@ jobs:
                       git tag ${GRAVITEEIO_VERSION_WITH_QUALIFIER}
 
                       # If we are on master branch, we need to create the support branch
-                      if [ "${CIRCLE_BRANCH}" = "master" ]; then
+                      if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
We need to be careful with that. I should work because the OS running the script in Ubuntu and Bash is used. But in POSIX Shell we need to use `=` 

It is not very clear. but I remember facing issue with that 🙈 

https://linuxize.com/post/how-to-compare-strings-in-bash/
https://queirozf.com/entries/posix-shell-tests-and-conditionals-examples-and-reference

Reply:
You're right. I reverted this part and fixed an already existing `==` syntax